crates/ruff_formatter/src/buffer.rs

```diff
@@ -122,7 +122,7 @@ impl BufferSnapshot {
                 Err(err) => {
                     panic!(
                         "Tried to unwrap snapshot of type {:?} as {:?}",
-                        err.type_id(),
+                        (*err).type_id(),
```

Comment:
Do you know which rule triggered here?

Reply:
I think it was calling `type_id` on `Box<dyn >`